python/setup.py

```diff
@@ -229,6 +229,7 @@ def get_lib_file_path(lib_name, backend=""):
     "pyyaml",
     "requests>=2.19.0",
     "scipy>=1.1.0",
+    "torch<2.4.0",
```

Comment:
why adding this? our nightly build has support 2.4.0 and next release will also support it

Reply:
Then we need to change it to 2.5?

Reply:
what is this change for?

Reply:
This change should be made in release branches.